.github/workflows/swift-toolchain.yml

```diff
@@ -648,6 +648,9 @@ jobs:
         with:
           python-version: '${{ env.PYTHON_VERSION }}'
 
+      - uses: nuget/setup-nuget@v2
+        if: matrix.arch == 'arm64'
```

Comment:
Interesting, why is this not needed on GHA? The matrix is for the cross-compilation target, so I'm not sure what this is.

Reply:
It's preinstalled on GHA hosted runner image via chocolatey: https://github.com/actions/runner-images/blob/main/images/windows/toolsets/toolset-2022.json#L329

Reply:
I'm not sure I understand why this is not needed when compiling for x64 though if the builder image is the same. Is there some other change that I'm missing that results in this being installed in that case or somehow does a different builder image get used?

Reply:
It's because of line 656 down below. The "nuget install pythonarm64" is only run `if: matrix.arch == 'arm64'`

Reply:
A comment to reflect that would be nice.